docs/source/guides/sql_database_integration.ipynb

```diff
@@ -0,0 +1,144 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# SQL Database Integration \n",
+    "\n",
+    "`featuretools_sql` is an add-on library to `featuretools` that supports automatic `EntitySet` creation from relational data stored in a relational database management system (RDBMS).\n",
+    "\n",
+    "Currently, `featuretools_sql` is compatible with the following RDBMS's:\n",
```

Comment:
<img width="688" alt="Screen Shot 2022-08-31 at 4 18 00 PM" src="https://user-images.githubusercontent.com/8726321/187774083-7555319a-1b3b-4272-8445-23283ccf801e.png">
List format does not show up correctly

Reply:
Interestingly it shows up fine for me in VSCode. I will rebuild and try again

Reply:
should work now